models/repo.go

```diff
@@ -990,13 +990,8 @@ func (repo *Repository) getUsersWithAccessMode(e Engine, mode AccessMode) (_ []*
 }
 
 // DescriptionHTML does special handles to description and return HTML string.
-func (repo *Repository) DescriptionHTML() template.HTML {
-	desc, err := markup.RenderDescriptionHTML([]byte(repo.Description), repo.HTMLURL(), repo.ComposeMetas())
-	if err != nil {
-		log.Error("Failed to render description for %s (ID: %d): %v", repo.Name, repo.ID, err)
-		return template.HTML(markup.Sanitize(repo.Description))
-	}
-	return template.HTML(markup.Sanitize(string(desc)))
+func (repo *Repository) DescriptionHTML() string {
+	return string(markdown.Render([]byte(repo.Description), repo.HTMLURL(), repo.ComposeMetas()))
```

Comment:
I think `markup.RenderDescriptionHTML` has done that.

Reply:
No, it's a little different with markdown.
example:
![image](https://user-images.githubusercontent.com/25342410/98017740-a3349200-1e3a-11eb-8a72-d765aef32d81.png)


 I wander why design this special ruler to render it.